src/test/java/space/obminyashka/items_exchange/controller/UserControllerTest.java

```diff
@@ -47,6 +44,7 @@
 @AutoConfigureMockMvc
 class UserControllerTest {
 
+    public static final String PATH_USER_INFO = "/user/info";
```

Comment:
All this quantity of constants become a mess. It would be better to create couple of root constants and to use them:
 public static final String USER_INFO = "/user/info";
 public static final String USER_SERVICE = "/user/service";
 public static final String PATH_USER_CHANGE_PASSWORD = USER_SERVICE + "/pass";
 ...